en/docs/install-and-setup/setup/deployment-best-practices/security-guidelines-for-production-deployment.md

```diff
@@ -255,6 +255,13 @@ been removed from Hotspot JVM.</p>
 <td><p>For password recovery, you can define a regular expression to validate the callback URL. The default configuration allows any callback URL. Note that if you are using the recovery option, it is highly recommended to define the regular expression that validates and only allows access to specific callback URLs.</p>
 </td>
 </tr>
+<tr class="odd">
+<td><p>Strict host verification</p>
+<p><br />
+</p></td>
+<td><p>To configure strict host verification, you can template the `site/public/conf/settings.js` file of each webapp - Publisher, Developer Portal, and Admin. You can template this file using a tool like Ansible if that is what you use now or you can maintain a template file for the settings.js alone as a customization where you can change it through the deployment.toml file.</p>
```

Comment:
Also shall we reword this to clearly give the path to the settings.js file in each webapp? Something like `site/public/conf/settings.js` in Publisher and Admin portal, `site/public/theme/settings.js` in Developer Portal.

Reply:
Fixed.